mappings/net/minecraft/entity/data/TrackedDataHandler.mapping

```diff
@@ -7,3 +7,22 @@ CLASS net/minecraft/class_2941 net/minecraft/entity/data/TrackedDataHandler
 	METHOD method_12716 read (Lnet/minecraft/class_2540;)Ljava/lang/Object;
 		ARG 1 buf
 	METHOD method_12717 create (I)Lnet/minecraft/class_2940;
+		ARG 1 id
+	METHOD method_43237 of (Lnet/minecraft/class_2359;)Lnet/minecraft/class_2941;
+		ARG 0 registry
+	METHOD method_43238 (Lnet/minecraft/class_2359;Lnet/minecraft/class_2540;)Ljava/lang/Object;
+		ARG 1 buf
+	METHOD method_43239 (Lnet/minecraft/class_2359;Lnet/minecraft/class_2540;Ljava/lang/Object;)V
+		ARG 1 buf
+		ARG 2 value
+	METHOD method_43240 of (Ljava/lang/Class;)Lnet/minecraft/class_2941;
```

Comment:
I suggest `ofEnum`, as they may create another `ofRecord` that takes a `Class` parameter as well. In that case, the `of` name will be confusing,